### PR TITLE
feat(Qwik): Add content for podcasts

### PR DESCRIPTION
**Pull request by SDaian**
## Type of change

Closes: #284 

- [X] Content addition
- [ ] Bug fix
- [ ] Behavior change

## Summary of change

- [X]  Research a list of Podcasts
- [X]  Add the Podcasts information in the data files
- [X]  Test each individual look and external link

## Checklist

- [X] The changes follow the [contributing guidelines](https://github.com/thisdot/framework.dev/blob/main/CONTRIBUTING.md)
- [X] I have searched all existing content and verified my additions are novel
      and unique
- [X] The content was added to the end of the appropriate data list
- [X] All required content fields are accurately populated
- [X] All items are tagged with all relevant tags

### CODE REVIEW

**Review by AllStackDev1 — COMMENTED** [review]
I think there is still more here https://qwik.builder.io/media/ scroll down to podcast

**Comment by JiaLiPassion** [comment]
Here I found some newest ones.
https://t.co/SXibYspR6E

**Review by AllStackDev1 — APPROVED** [review]

**Review by JiaLiPassion — APPROVED** [review]

File: packages/site/src/data/qwik/podcasts.ts [hunk 1]
```diff
@@ -1,16 +1,77 @@
...
+		hosts: ["Shai Reznik - Misko Hevery"],
+		description:
+			"Have you seen Qwik in action yet? If not, here's your chance! We've invited the Father of Qwik, aka the Father of Angular, Misko Hevery himself to show us what he built. As a HUGE bonus, we also invited one of our favorite teachers, Shai Reznik of HiRez.io fame. Sit back, buckle up, and prepare for a wild ride into the future of VERY FAST web pages!.",
+		rss: "https://www.youtube.com/watch?v=iJZaT-AvJ-o",
+		href: "https://www.youtube.com/watch?v=iJZaT-AvJ-o",
+		tags: ["qwik"],
+	},
+	{
+		title: "JS Poland",
+		image: "	https://i3.ytimg.com/vi/7MgNMIPISY4/hqdefault.jpg",
```
Comment (honzikec) [inline packages/site/src/data/qwik/podcasts.ts:39]:
```suggestion
		image: "https://i3.ytimg.com/vi/7MgNMIPISY4/hqdefault.jpg",
```

**Review by honzikec — APPROVED** [review]
Looks good, just found one extra space 🙂

**Pull request merged**